prow/plugins/trigger/trigger.go

```diff
@@ -184,6 +185,46 @@ func TrustedUser(ghc trustedUserClient, trigger *plugins.Trigger, user, org, rep
 	return member, nil
 }
 
+func skippedStatusFor(context string) github.Status {
+	return github.Status{
+		State:       github.StatusSuccess,
+		Context:     context,
+		Description: "Skipped.",
+	}
+}
+
+// runAndSkipJobs executes the config.Presubmits that are requested and posts skipped statuses
+// for the reporting jobs that are skipped
+func runAndSkipJobs(c Client, pr *github.PullRequest, requestedJobs []config.Presubmit, skippedJobs []config.Presubmit, eventGUID string, elideSkippedContexts bool) error {
+	if err := validateContextOverlap(requestedJobs, skippedJobs); err != nil {
+		return err
```

Comment:
:+1: It will be nice to see an immediate error in the logs if this somehow happens instead of depending on a user to notice it.